translations/de.yml

```diff
@@ -0,0 +1,129 @@
+---
+de:
+  title: "Bitcoin Cash - Peer-to-Peer Electronic Cash"
+  meta:
+    description: |
+      Bitcoin Cash bringt sicheres Geld. H�ndler und Kunden werden durch geringe Geb�hren und zuverl�ssige Best�tigungen erm�chtigt. Die Zukunft leuchtet hell mit ungehemmten Wachstum, weltweiter Akzeptanz, ungehinderter Innovation und dezentraler Entwicklung.
+    twitter:
+      title: "Bitcoin Cash - Peer-to-Peer Electronic Cash"
+      description: |
+      Bitcoin Cash bringt sicheres Geld. H�ndler und Kunden werden durch geringe Geb�hren und zuverl�ssige Best�tigungen erm�chtigt. Die Zukunft leuchtet hell mit ungehemmten Wachstum, weltweiter Akzeptanz, ungehinderter Innovation und dezentraler Entwicklung.
+    og:
+      title: "Bitcoin Cash - Peer-to-Peer Electronic Cash"
+      description: |
+      Bitcoin Cash bringt sicheres Geld. H�ndler und Kunden werden durch geringe Geb�hren und zuverl�ssige Best�tigungen erm�chtigt. Die Zukunft leuchtet hell mit ungehemmten Wachstum, weltweiter Akzeptanz, ungehinderter Innovation und dezentraler Entwicklung.
+    misc:
+      logo: "logo"
+    sections:
+      about: "�ber bitcoincash.org"
+      features: "Features"
+      nodes: "Nodes"
+      wallets: "Wallets"
+      community: "Community"
+      services: "Services"
+      projects: "Projekte"
+      exchanges: "B�rsen"
+      faq: "FAQ"
+    layout:
+      bitcoin_cash: "Bitcoin Cash"
+      lead: "Peer-to-Peer Electronic Cash"
```

Comment:
bitcoin.de hat in der Uebersetzung des Whitepapers "Elektronisches Peer-to-Peer-Bezahlsystem" verwendet.

https://www.bitcoin.de/de/bitcoin-whitepaper-deutsch-html